test/expected/extension.out

```diff
@@ -1,80 +1,218 @@
+ _timescaledb_internal.generate_uuid()
+ _timescaledb_internal.get_chunk_colstats(regclass)
+ _timescaledb_internal.get_chunk_relstats(regclass)
+ _timescaledb_internal.get_create_command(name)
+ _timescaledb_internal.get_git_commit()
+ _timescaledb_internal.get_os_info()
+ _timescaledb_internal.get_partition_for_key(anyelement)
+ _timescaledb_internal.get_partition_hash(anyelement)
+ _timescaledb_internal.get_time_type(integer)
+ _timescaledb_internal.hist_combinefunc(internal,internal)
+ _timescaledb_internal.hist_deserializefunc(bytea,internal)
+ _timescaledb_internal.hist_finalfunc(internal,double precision,double precision,double precision,integer)
+ _timescaledb_internal.hist_serializefunc(internal)
+ _timescaledb_internal.hist_sfunc(internal,double precision,double precision,double precision,integer)
+ _timescaledb_internal.hypertable_constraint_add_table_fk_constraint(name,name,name,integer)
+ _timescaledb_internal.hypertable_from_main_table(regclass)
+ _timescaledb_internal.hypertable_invalidation_log_delete(integer)
+ _timescaledb_internal.hypertable_local_size(name,name)
+ _timescaledb_internal.hypertable_remote_size(name,name)
+ _timescaledb_internal.indexes_local_size(name,name)
+ _timescaledb_internal.indexes_remote_size(name,name,name)
+ _timescaledb_internal.insert_blocker()
+ _timescaledb_internal.interval_to_usec(interval)
+ _timescaledb_internal.invalidation_cagg_log_add_entry(integer,bigint,bigint)
+ _timescaledb_internal.invalidation_hyper_log_add_entry(integer,bigint,bigint)
+ _timescaledb_internal.invalidation_process_cagg_log(integer,integer,regtype,bigint,bigint,integer[],bigint[],bigint[])
+ _timescaledb_internal.invalidation_process_cagg_log(integer,integer,regtype,bigint,bigint,integer[],bigint[],bigint[],text[])
+ _timescaledb_internal.invalidation_process_hypertable_log(integer,integer,regtype,integer[],bigint[],bigint[])
+ _timescaledb_internal.invalidation_process_hypertable_log(integer,integer,regtype,integer[],bigint[],bigint[],text[])
+ _timescaledb_internal.is_main_table(name,name)
+ _timescaledb_internal.is_main_table(regclass)
+ _timescaledb_internal.last_combinefunc(internal,internal)
+ _timescaledb_internal.last_sfunc(internal,anyelement,"any")
+ _timescaledb_internal.main_table_from_hypertable(integer)
+ _timescaledb_internal.materialization_invalidation_log_delete(integer)
+ _timescaledb_internal.partialize_agg(anyelement)
+ _timescaledb_internal.ping_data_node(name)
+ _timescaledb_internal.policy_compression_execute(integer,integer,anyelement,integer,boolean,boolean)
+ _timescaledb_internal.policy_compression(integer,jsonb)
+ _timescaledb_internal.policy_recompression(integer,jsonb)
+ _timescaledb_internal.policy_refresh_continuous_aggregate(integer,jsonb)
+ _timescaledb_internal.policy_reorder(integer,jsonb)
+ _timescaledb_internal.policy_retention(integer,jsonb)
+ _timescaledb_internal.process_ddl_event()
+ _timescaledb_internal.range_value_to_pretty(bigint,regtype)
```

Comment:
```suggestion
 _timescaledb_internal.range_value_to_pretty(bigint,regtype)
 _timescaledb_internal.relation_size(regclass)
```

Seems you forgot to include this function introduced by 33bbdccdcd7ad805c3463ba6426dea0510497ed1

Reply:
hadn't rebased, fixed now